README.md

```diff
@@ -30,6 +32,36 @@ As framework we use [🅱️ Bootstrap](https://getbootstrap.com) which you migh
 - SVG images can be compressed using the also lossless scour using the command line: `scour -i filename.svg -o filename-compressed.svg --create-groups --enable-id-stripping --enable-comment-stripping --shorten-ids --remove-metadata --strip-xml-prolog --no-line-breaks;`
 
 
+## Contributors
+
+This project exists thanks to all the people who contribute. 
+<a href="graphs/contributors"><img src="https://opencollective.com/opensourcediversity/contributors.svg?width=890&button=false" /></a>
+
+
+## Backers
+
+Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/opensourcediversity#backer)]
+
+<a href="https://opencollective.com/opensourcediversity#backers" target="_blank"><img src="https://opencollective.com/opensourcediversity/backers.svg?width=890"></a>
+
+
+## Sponsors
+
+Support this project by becoming a sponsor. Your logo will show up here with a link to your website. [[Become a sponsor](https://opencollective.com/opensourcediversity#sponsor)]
+
+<a href="https://opencollective.com/opensourcediversity/sponsor/0/website" target="_blank"><img src="https://opencollective.com/opensourcediversity/sponsor/0/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcediversity/sponsor/1/website" target="_blank"><img src="https://opencollective.com/opensourcediversity/sponsor/1/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcediversity/sponsor/2/website" target="_blank"><img src="https://opencollective.com/opensourcediversity/sponsor/2/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcediversity/sponsor/3/website" target="_blank"><img src="https://opencollective.com/opensourcediversity/sponsor/3/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcediversity/sponsor/4/website" target="_blank"><img src="https://opencollective.com/opensourcediversity/sponsor/4/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcediversity/sponsor/5/website" target="_blank"><img src="https://opencollective.com/opensourcediversity/sponsor/5/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcediversity/sponsor/6/website" target="_blank"><img src="https://opencollective.com/opensourcediversity/sponsor/6/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcediversity/sponsor/7/website" target="_blank"><img src="https://opencollective.com/opensourcediversity/sponsor/7/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcediversity/sponsor/8/website" target="_blank"><img src="https://opencollective.com/opensourcediversity/sponsor/8/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcediversity/sponsor/9/website" target="_blank"><img src="https://opencollective.com/opensourcediversity/sponsor/9/avatar.svg"></a>
```

Comment:
Does anyone know what these links and images are supposed to do?

Reply:
I am asking since there is an asymmetry between the way the Contributors and Backers have been added (with a single img) compared to 9 link + img.

Reply:
You can check the preview/diff to see how it looks like :) – it shows the avatars of the sponsors https://github.com/opensourcediversity/opensourcediversity.org/pull/45/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8

Reply:
Not sure why the assymetry exists, might just be that it’s a work in progress.

Reply:
I checked the preview, but I don't see any avatar. Maybe because we don't have sponsors. Do you see anything there? :)